src/commands/LsCommand.js

```diff
@@ -1,15 +1,30 @@
 import Command from "../Command";
+import PackageUtilities from "../PackageUtilities";
 
 export default class LsCommand extends Command {
   initialize(callback) {
-    // Nothing to do...
+    if (this.input.length) {
```

Comment:
I'm not sure why you need to do all this work. You should already have the package graph which will allow you to find the tree of dependencies of a single package.